src/applications/disability-benefits/686/containers/ConfirmationPage.jsx

```diff
@@ -83,21 +84,7 @@ export class ConfirmationPage extends React.Component {
             </ul>
           )}
         </div>
-        <AskVAQuestions>
-          <p className="help-talk">Enrollment or Eligibility questions:</p>
-          <p className="help-phone-number">
-            <a className="help-phone-number-link" href="tel:+1-877-222-8387">
-              1-877-222-8387
-            </a>
-            <br />
-            TTY:{' '}
-            <a className="help-phone-number-link" href="tel:+18008778339">
-              1-800-877-8339
-            </a>
-            <br />
-            Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET)
-          </p>
-        </AskVAQuestions>
+        <FormFooter formConfig={{ getHelp: GetFormHelp }} />
```

Comment:
It looks like the FormFooter component displays null on confirmation pages (by checking the pathname). If so, would we still want to include it here? Or perhaps update the component to display on confirmation pages?

Reply:
It's only not displaying on confirmation pages when you pass the current location and you're on the confirmation page. In this case, we're not passing that, so it is rendered here (and the one from `footerContent` is not). I'm not exactly sure why some of our forms work this way (and it's only some, not all of them). I think it's worth going back and seeing if we can adjust that, but I didn't want to make more changes in this PR.